pyproject.toml

```diff
@@ -0,0 +1,60 @@
+[build-system]
+requires = ["hatchling"]
+build-backend = "hatchling.build"
+
+[project]
+name = "analysis-dashboard"
+version = "0.1.0"
```

Comment:
we should make this dynamic bassed on git tags, but this could always happen in a later PR